### ISSUE TITLE
No errors, form doesn't submit with JsValidator::formRequest

### ISSUE THREAD

**Issue by aewing** [proengsoft/laravel-jsvalidation#81]
I'm using JsValidator::formRequest to validate my forms using custom request classes.
My rules:
````php
        return [
            'code' => 'required|unique|min:4|max:16',
            'type' => 'required',
            'amount' => 'required'
        ];
````

Validation seems to be working, I get errors when I should, and no errors when I shouldn't, but the form never submits.

**Comment by aewing** [comment]
There was an error in my Request class. Resolving this error allowed my form to submit, but it seems that there should be some sort of alert or error notification here.

**Comment by torrentalle** [comment]
This is fixed  in *dev-master* branch 1e86f0157ba8bb172b6e5245c9f262cb1e59895e.
I'm testing an documenting new release.

**Comment by torrentalle** [comment]
This feature is published to [v1.2.0rc1](https://github.com/proengsoft/laravel-jsvalidation/tree/v1.2.0rc1) pre-release.
I'm preparing the documentation before to launch the production release
